styles.css

```diff
@@ -1001,28 +1002,60 @@ body {
 
 .form-input {
   padding: 12px 16px 12px 0;
-  outline: none;
   border-style: none;
-  border-bottom: 1px solid #dbd8d7;
+  outline: #979493 solid 1px;
+  border-radius: 5px;
   margin-bottom: 39px;
   width: 100%;
   background-color: #e5e5e5;
 }
 
+.form-input:focus {
+  outline: none;
+  border-bottom: 3px solid #ee8549;
+}
+
+.form-input#email:valid {
+  outline: none;
+  border: none;
+  box-shadow: -2px 5px 10px 2px #488357;
+}
+
+.border-red {
+  outline: none;
+  border: none;
+  box-shadow: -2px 5px 10px 2px #db3737;
+}
+
+.border-normal {
+  outline: none;
+  box-shadow: none;
+  border-bottom: 3px solid #ee8549;
+}
+
+/* .border-green {
+  outline: none;
+  border: none;
+  box-shadow: -2px 5px 10px 2px #488357;
+}  */
```

Comment:
- [ ] It will be nice to delete unused codes from your codebase. Leaving it makes your codebase looks unreadable and untidy. You can use git history if you like to see previous codes.